docker-compose.yml

```diff
@@ -28,6 +28,10 @@ services:
       CONCOURSE_EXTERNAL_URL: http://localhost:8080
       CONCOURSE_ADD_LOCAL_USER: test:test,guest:guest
       CONCOURSE_MAIN_TEAM_LOCAL_USER: test
+      CONCOURSE_ENABLE_BUILD_AUDITING: 'true'
+      CONCOURSE_ENABLE_JOB_AUDITING: 'true'
+      CONCOURSE_ENABLE_PIPELINE_AUDITING: 'true'
+      CONCOURSE_ENABLE_TEAM_AUDITING: 'true'
```

Comment:
Would you mind backing this change out? I don't think it's necessary as a dev default. Guessing this was checked in from local testing.

Reply:
For sure, I can pull this out, good catch.